README.md

```diff
@@ -2093,53 +2299,73 @@ In addition to the **y1** and **y2** output channels, Plot.stackY computes a **y
 
 If two arguments are passed to the stack transform functions below, the stack-specific options (**offset**, **order**, and **reverse**) are pulled exclusively from the first *options* argument, while any channels (*e.g.*, **x**, **y**, and **z**) are pulled from second *options* argument. Options from the second argument that are not consumed by the stack transform will be passed through. Using two arguments is sometimes necessary is disambiguate the option recipient when chaining transforms.
 
-#### Plot.stackY([*stack*, ]*options*)
+<!-- jsdoc stackY -->
+
+#### Plot.stackY(*stackOptions*, *options*)
```

Comment:
renamed argument; and let's make a note that the optional argument is not shown with brackets (but the example below reminds the user that it's quite possible to use only one argument).

Reply:
same here, the code uses `stackOptions` as the param name to avoid shadowing the `stack` function.

Reply:
hmmm, OK… but this also changes the link's name (produced automatically by github) from #plotstackystack-options to #plotstackystackoptions-options; it's referenced below.